### PR TITLE
missing colon and a bit of formatting

### PR DESCRIPTION
**Pull request by opakholis**

### CODE REVIEW

**Review by zbirenbaum — APPROVED** [review]
I don't see a missing colon added anywhere, but the formatting fixes are nice.

**Comment by zbirenbaum** [comment]
In the future, I'll probably hold off merging on changes like this until updating the api and then squash them, but since I don't have a timeline on that yet I'll accept this.

File: README.md [hunk 1]
```diff
@@ -11,28 +11,31 @@ Copilot suggestions will automatically be loaded into your cmp menu as snippets
...
     "zbirenbaum/copilot-cmp",
-    after = {"copilot.lua", "nvim-cmp"},
+    after = { "copilot.lua", "nvim-cmp" },
 }
 ```
+
 If you do not have copilot.lua installed, go to https://github.com/zbirenbaum/copilot.lua and follow the instructions there before installing this one
 
 ### Configuration
-To link cmp with this source, simply go into your cmp configuration file and include `{name = "copilot}` under your sources
```
Comment (opakholis) [inline README.md:23]: the inline code here I mean, it's not really necessary, but I was itching to see it.. so I did it

**Comment by opakholis** [comment]
> In the future, I'll probably hold off merging on changes like this until updating the api and then squash them, but since I don't have a timeline on that yet I'll accept this.

I really love this plugin. of course, I'm waiting for the future. thanks anyway!

File: README.md (hunk 1 above)
Comment (zbirenbaum) [reply README.md:23]: Oh man I completely missed that. I totally see how that would be anxiety inducing, I have some now just looking at it 😂

**Comment by zbirenbaum** [comment]
> > In the future, I'll probably hold off merging on changes like this until updating the api and then squash them, but since I don't have a timeline on that yet I'll accept this.
> 
> I really love this plugin. of course, I'm waiting for the future. thanks anyway!

I’m really glad you like it, I haven’t had too much feedback so far outside of issue posts so it means a lot to hear this!

**Pull request merged**